src/core/server/saved_objects/migrationsv2/actions/index.ts

```diff
@@ -419,6 +420,133 @@ export const pickupUpdatedMappings = (
     .catch(catchRetryableEsClientErrors);
 };
 
+/** @internal */
+export interface OpenPitResponse {
+  pitId: string;
+}
+
+// how long ES should keep PIT alive
+const pitKeepAlive = '10m';
```

Comment:
When I tested performance impact, migration of `1000` of SO took ~1sec (on MacBook Pro). Considering that we allow users to specify `batchSize` and they can run Kibana on slower hardware, I picked something that definitely would enough for a single batch of transformation. 
I would not make it a configurable value to avoid bloating the Kibana public contract.